pydra/engine/core.py

```diff
@@ -183,6 +184,8 @@ def __init__(
         self.cache_locations = cache_locations
         self.allow_cache_override = True
         self._checksum = None
+        self._uid = None
+        self._uid_states = None
```

Comment:
still worried about this as this now generates a uuid for every split and stores it in the task. perhaps this can be dynamically generated at the point of need.

Reply:
you're right, this is pretty useless - right now it's used only in `run_el`, removed